test/e2e/selectors/selectors_test.go

```diff
@@ -186,56 +200,73 @@ var _ = Describe("Field Selectors in Flight Control", Ordered, func() {
+			out, err = RunGetDevices(harness, "--field-selector", "unsupported.field")
 			Expect(err).To(HaveOccurred())
 			Expect(out).To(ContainSubstring(unknownSelector))
 
 			By("returns an error for an invalid field selector syntax")
-			out, err = harness.CLI("get", "devices", "--field-selector", "metadata.name@=device1-name")
+			out, err = RunGetDevices(harness, "--field-selector", "metadata.name@=device1-name")
 			Expect(err).To(HaveOccurred())
 			Expect(out).To(ContainSubstring(failedToParse))
 
 			By("returns an error for an incorrect field type")
-			out, err = harness.CLI("get", "devices", "--field-selector", "metadata.name>10")
+			out, err = RunGetDevices(harness, "--field-selector", "metadata.name>10")
 			Expect(err).To(HaveOccurred())
 			Expect(out).To(ContainSubstring("unsupported for type string"))
 			Expect(out).To(ContainSubstring("metadata.name"))
 
 			By("returns an error for filtering devices by deprecated contains operator")
-			out, err = harness.CLI("get", "devices", "--field-selector", "metadata.labels contains region=eu-west-1")
+			out, err = RunGetDevices(harness, "--field-selector", "metadata.labels contains region=eu-west-1")
 			Expect(err).To(HaveOccurred())
 			Expect(out).To(ContainSubstring("field is marked as private and cannot be selected"))
 
 			By("returns an error for filtering devices by deprecated owner flag")
-			out, err = harness.CLI("get", "devices", "--owner")
+			out, err = RunGetDevices(harness, "--owner")
 			Expect(err).To(HaveOccurred())
 			Expect(out).To(ContainSubstring(unknownFlag))
 
 			By("returns an error for filtering devices by deprecated status-filter flag")
-			out, err = harness.CLI("get", "devices", "--status-filter=updated.status=UpToDate")
+			out, err = RunGetDevices(harness, "--status-filter=updated.status=UpToDate")
 			Expect(err).To(HaveOccurred())
 			Expect(out).To(ContainSubstring(unknownFlag))
 
 			By("returns an error for bad syntax using =! operator")
-			out, err = harness.CLI("get", "devices", "-l", "'region=!eu-west-1'")
+			out, err = RunGetDevices(harness, "-l", "'region=!eu-west-1'")
 			Expect(err).To(HaveOccurred())
 			Expect(out).To(ContainSubstring(failedToParse))
 
 			By("returns an error for bad syntax using ! operator outside the quotes")
-			out, err = harness.CLI("get", "devices", "-l", "!'region=eu-west-1'")
+			out, err = RunGetDevices(harness, "-l", "!'region=eu-west-1'")
 			Expect(err).To(HaveOccurred())
 			Expect(out).To(ContainSubstring(failedToParse))
 
 			By("returns an error for bad syntax using ! operator at the start of the quotes")
-			out, err = harness.CLI("get", "devices", "-l", "'!region=eu-west-1'")
+			out, err = RunGetDevices(harness, "-l", "'!region=eu-west-1'")
 			Expect(err).To(HaveOccurred())
 			Expect(out).To(ContainSubstring(failedToParse))
-
 		})
 	})
 })
+
+// RunGetDevices executes "get devices" CLI command with optional arguments.
+func RunGetDevices(harness *e2e.Harness, args ...string) (string, error) {
```

Comment:
The function is general for the devices and could be used by others, please, move it under util folder